mypyc/ir/rtypes.py

```diff
@@ -234,6 +234,10 @@ def __repr__(self) -> str:
 short_int_rprimitive = RPrimitive('short_int', is_unboxed=True, is_refcounted=False,
                                   ctype='CPyTagged')  # type: Final
 
+# low level integer (corresponds to C's 'int').
+c_int_rprimitive = RPrimitive('c_int', is_unboxed=True, is_refcounted=False,
+                              ctype='int')  # type: Final
```

Comment:
The ctype should be `Py_ssize_t`, since we want the integer to have the same size as a pointer.